src/TodoListItem.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+function TodoListItem(props) {
+  return (
+    <li key={props.key}>{props.todo.title}</li>
```

Comment:
List item, in this context, does not need a key as this component does not create multiple elements. Its parent, TodoList is where the key needs to be and you already have that in place. Please remove this one.

Reply:
Ah ok! Got it. Thank you!